### PR TITLE
add header and basic location list

### PR DESCRIPTION
**Pull request by aaaleeenaaa**
Add header for basic structure, location list component and card component

Many cards so that it can be seen that the header stays visible while scrolling

### CODE REVIEW

File: components/Card.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  padding: 1em 3em;
+  margin: 15px 0px;
+  text-align: center;
+  border: 3px grey solid;
+  border-radius: 10px;
+`;
+
+const StyledLink = styled.a`
+  text-decoration: none;
+`;
```
Comment (NikRoe) [inline components/Card.js]: you should use the Link component from next instead (don't forget to import it in your file)

```suggestion
const StyledLink = styled(Link)`
  text-decoration: none;
`;
```

File: components/Header.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+const Headline = styled.h1`
```
Comment (NikRoe) [inline components/Header.js]: if this is going to be a Header, you could also thing about using the header element to be more semantic:

```suggestion
const Headline = styled.header`
```

File: components/Layout.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import Header from "./Header.js";
+import styled from "styled-components";
+import Head from "next/head.js";
```
Comment (NikRoe) [inline components/Layout.js]: you don't need ".js" here

```suggestion
import Head from "next/head";
```

File: components/LocationList.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      <Card />
+      <Card />
+      <Card />
+      <Card />
+      <Card />
+      <Card />
+      <Card />
+      <Card />
+      <Card />
+      <Card />
```
Comment (NikRoe) [inline components/LocationList.js:25]: fine as a starting point, but as soon as possible, you should make this part of the code dynamic :)

**Review by NikRoe — CHANGES_REQUESTED** [review]
Hi Alena,

this looks very nice and clean.

I left only a few notes and comments. Please take a look at those and implement my suggestions :)

File: components/LocationList.js (hunk 4 above)
Comment (aaaleeenaaa) [reply components/LocationList.js:25]: Yes, that is part of the next user story! This list is only a placeholder for the beginning.

**Review by NikRoe — APPROVED** [review]
LGTM 👍

**Pull request merged**